src/components/GlobalData/index.js

```diff
@@ -33,6 +35,10 @@ export default class GlobalData extends React.Component {
       this.setState({ isLoading: false, hasError: true });
     }
   };
+  getCurrencyValue = (key) => {
+    const currency = this.props.currency;
```

Comment:
destructure

Reply:
const { currency } = this.props; like so yes?